tracpro/templates/baseline/baselineterm_data.html

```diff
@@ -0,0 +1,147 @@
+{% extends "smartmin/base.html" %}
+{% load smartmin i18n compress %}
+
+{% block content %}
+
+    {% if form.errors %}
+        <div class="alert alert-danger form-errors">
+            Please correct the errors below.
+        </div>
+    {% endif %}
+
+    {% if form.non_field_errors %}
+        <ul class="field-errors">
+            <li>{{ form.non_field_errors }}</li>
+        </ul>
+    {% endif %}
+
+    <div>
+      <form id="spoofform" class="smartmin-form form-horizontal ng-pristine ng-valid" action="." method="post">
```

Comment:
Smartmin has some cool tools for displaying bootstrap forms. This template powers the create and update views: https://github.com/nyaruka/smartmin/blob/master/smartmin/templates/smartmin/form.html